### PR TITLE
Copy updates to kubeflow install page

### LINKED ISSUE #7525: [COPY UPDATE] install - "Install Kubeflow on Microk8s"
Please update install - "Install Kubeflow on Microk8s" from the [copy doc](https://docs.google.com/document/d/1KVyyvZsDyrfoi5tK2m5SLWs9YuHg9hgqVz0c8cWHsf8/edit?usp=drivesdk)

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- updated kubeflow install instructions

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubeflow/install
- Compare it to the [copy](https://docs.google.com/document/d/1KVyyvZsDyrfoi5tK2m5SLWs9YuHg9hgqVz0c8cWHsf8/edit#)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7525

### CODE REVIEW

**Comment by deadlight** [comment]
@anthonydillon ah, I see that there were more changes than the ones flagged. I'll do a full check.

**Comment by deadlight** [comment]
@anthonydillon ready for another review

File: templates/kubeflow/install.html [hunk 1]
```diff
@@ -31,15 +31,21 @@ <h1>Install Kubeflow on Ubuntu</h1>
...
     <ol class="p-stepped-list--detailed">
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Requirements</h3>
+        <div class="p-stepped-list__content">
+          <p>In order to install kubeflow you will need an Ubuntu machine with 16Gb of RAM and 50Gb of free disk. </p>
+        </div>
+      </li>
 
       <li class="p-stepped-list__item">
         <h3 class="p-stepped-list__title">Install MicroK8s</h3>
```
Comment (anthonydillon) [inline templates/kubeflow/install.html:51]: The `p-stepped-list__content` div is not closed.

File: templates/kubeflow/install.html [hunk 2]
```diff
@@ -89,40 +83,34 @@ <h3 class="p-stepped-list__title">Enable Kubeflow</h3>
...
             <p>
-              If you installed MicroK8s on your local host, then you can use localhost as the IP address in your browser.
-            </p>
-            <p>
-              Otherwise, if you used Multipass as per the instructions above, you can get the IP address of the VM with either <code>multipass list</code> or <code>multipass info kubeflow</code>.
+              After installing kubeflow, an IP address and credentials will be prompted on your terminal. 
             </p>
             <p>
-              Now you should go to your browser and point browser to either:
+              If you installed Microk8s on your local host, you simply need to access the IP address provided on your browser: <code>https://[kubeflow dashboard IP]</kubeflow></code>.
```
Comment (anthonydillon) [inline templates/kubeflow/install.html:97]: Closing `</kubeflow>`

Comment (deadlight) [reply templates/kubeflow/install.html:97]: @anthonydillon can you tell that I'm using a new editor?

**Pull request merged**